src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,8 +1,23 @@
-export const TodoFilter = () => (
+import React from 'react';
+
+type Props = {
+  filterBy: string,
+  setFilterBy: (option: string) => void,
+  value: string,
+  setValue: (query: string) => void,
+};
+
+export const TodoFilter: React.FC<Props> = React.memo(({
+  filterBy, setFilterBy, value, setValue,
+}) => (
   <form className="field has-addons">
     <p className="control">
       <span className="select">
-        <select data-cy="statusSelect">
+        <select
+          data-cy="statusSelect"
+          value={filterBy}
+          onChange={(event) => setFilterBy(event.target.value)}
```

Comment:
it's better to move it to a separate function instead of using an anonymous function (just FYI)